README.rst

```diff
@@ -32,6 +32,26 @@ settings.
    add ``"google_maps"`` to the policy value list.
 3. Click the "Save changes" button.
 
+Adding Google Maps API key
+--------------------------
+
+This Xblock requires a Google Maps API key to work properly. Visit
+https://developers.google.com/maps/documentation/javascript/get-api-key
+for more instructions about Google Maps API keys.
+
+After acquiring an API key enable the following APIs in Google Developers Console:
+    1. Google Maps Embed API
+    2. Google Maps JavaScript API
+
+Include the API key in cms.env.json file to enable studio editing and
+lms.env.json file for student preview.
+Example:
+    "XBLOCK_SETTINGS": {
+        "GoogleMapsXBlock": {
+            "API_KEY": "YOUR API KEY GOES HERE"
+        }
+    },
+
```

Comment:
Add the following text after the example:
`Replace "YOUR API KEY GOES HERE" with the API key.`